eng/targets/ResolveReferences.targets

```diff
@@ -64,6 +62,16 @@
   </ItemDefinitionGroup>
 
   <ItemGroup Condition="'$(EnableCustomReferenceResolution)' == 'true'">
+    <!--
+      Add compile-only reference to Microsoft.Internal.Extensions.Refs to resolve the reference assemblies from the
+      Extensions repo.
+    -->
+    <PackageReference Include="Microsoft.Internal.Extensions.Refs"
+        Version="$(MicrosoftInternalExtensionsRefsPackageVersion)"
+        Condition=" '$(MicrosoftInternalExtensionsRefsPath)' == '' "
+        IsImplicitlyDefined="true"
+        PrivateAssets="All" />
```

Comment:
Will move this into the `ResolveCustomReferences` target to suppress warnings about `@(PackageReference)` items in our projects.